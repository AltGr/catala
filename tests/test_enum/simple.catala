@Article@

/*
new enum E:
  -- Case1 type int
  -- Case2

new scope A:
  param x type E
  param y type int 

scope A:
  def x := Case1 of 2
  def y := match x with 
    -- Case1 of i : 42
    -- Case2 : 43
*/