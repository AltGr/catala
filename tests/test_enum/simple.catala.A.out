[RESULT] x -> Case1 2
[RESULT] y -> 42
